src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -39,8 +39,9 @@
 #include <set>
 #include <string>
 #include <algorithm>
-#include <utility>
 #include <map>
+#include <memory>
+#include <utility>
```

Comment:
@IGapchuk is memory header needed here?

Reply:
@mked-luxoft  Fixed in 36b0eadff97e5756477026c3d51f33e575f5d77a.